### DISCUSSION TITLE
Using nvtop for AMD GPUs

### DISCUSSION THREAD

**Discussion by TheNoise2Signal** [Syllo/nvtop#317] [Q&A]
I'm using a Linux server with kernel version `5.15.0-107-generic`. Have rocm-smi installed

I installed nvtop using `apt install nvtop`. But I'm not able to set the device version. 
Running nvtop right now, gives `No GPU to monitor.`.

Thank you

**Comment by TheNoise2Signal** [comment]
After removing the package (apt remove) and reinstalling using the PPA, see below
```
sudo add-apt-repository ppa:flexiondotorg/nvtop
sudo apt install nvtop
```
I get a new error. Still seems that the gpu isn't being detected
```
nvtop: ./src/extract_gpuinfo_amdgpu.c:324: initDeviceSysfsPaths: Assertion `gpu_info->hwmonDevice != NULL' failed.
Aborted (core dumped)
```

**Comment by austinbutler** [comment]
https://github.com/Syllo/nvtop/issues/250
